### ISSUE TITLE
Lloyd Wood #2 : channel and link coding out of the scope of the document

### ISSUE THREAD

**Issue by NicoKos** [irtf-nwcrg/draft-irtf-nwcrg-network-coding-satellites#47]
I'm puzzled as to how this document claims that channel and link codings are out of scope - and then talks about channel mode, dealing with varying channel conditions (yet without using channel codings), etc.
They're stated as out of scope...

**Comment by NicoKos** [comment]
If the intent is treating the overall end-to-end path as some sort of virtual 'channel' that coding packets are applied to, this needs to be said explicitly upfront. If 'channel' means more than one concatenated physical path here, this needs to be said. Unless your audience is only network coding people... the title should likely be '_network_ coding applicability to satellite-based scenarios' or similar, too. Needs to be really specific for clarity.

**Comment by NicoKos** [comment]
In PR #52 , we have changed: 
"Channel and link codings are gathered in the PHY layer coding and are out of the scope of this document."
by
"Channel and link codings are gathered in the PHY layer coding and are out of the scope of this document. It focuses on situations where coding is not widely deployed in current SATCOM systems."

For the comment on the end-to-end path, the notion of virtual 'channel' depends on the use-case and coding is not applied at the same level. 

For the title, we have moved from "Network coding and satellites" to "Coding techniques for satellite systems" from version 04 and 05 of the document to be more generic. I guess we should not go back to the "network coding"-like title.
